src/lib/components/workflow/workflows-summary-configurable-table/table-empty-state.svelte

```diff
@@ -0,0 +1,83 @@
+<script>
+  import { page } from '$app/stores';
+  import Icon from '$lib/holocene/icon/icon.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { workflowError } from '$lib/stores/workflows';
+  import emptyImage from '$lib/vendor/empty-state-dark_2x.png';
+  import noResultsImages from '$lib/vendor/empty-state-light_2x.png';
+
+  $: query = $page.url.searchParams.get('query');
+
+  const samples = [
+    'samples-go',
+    'samples-java',
+    'samples-typescript',
+    'samples-python',
+    'samples-dotnet',
+    'samples-php',
+  ];
+</script>
+
+<div
```

Comment:
We might want to consider adding an [aria-live="polite"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live) to the empty state to improve accessibility.

Reply:
👍